middleware/interpolate.js

```diff
@@ -1,3 +1,5 @@
+/*jshint esversion: 9 */
```

Comment:
What happens if you set this in `.jshintrc` instead? Does it complain about a bunch of our other code that is now a little bit of an "older" style?

Reply:
turns out it's fine